cpp/src/plasma/client.cc

```diff
@@ -308,7 +290,6 @@ PlasmaClient::Impl::~Impl() {}
 uint8_t* PlasmaClient::Impl::LookupOrMmap(int fd, int store_fd_val, int64_t map_size) {
   auto entry = mmap_table_.find(store_fd_val);
   if (entry != mmap_table_.end()) {
-    close(fd);
```

Comment:
file descriptor is not send any more if it had already been send, so we shouldn't close it here

Reply:
Should we close all the file descriptors in the PlasmaClient destructor?

Reply:
they have already been closed by the first LookupOrMmap call (see the close call below)